flink-ai-flow/ai_flow/ai_graph/ai_graph.py

```diff
@@ -35,6 +36,7 @@ class AIGraph(Graph):
     def __init__(self) -> None:
         super().__init__()
         self.nodes: Dict[Text, AINode] = {}
+        self._context_extractor: ContextExtractor = None
```

Comment:
The leading underscore seems to break naming conventions of this class

Reply:
The leading underscore indicates that the context_extractor is a private field.